kubesae/pod.py

```diff
@@ -28,9 +28,19 @@ def debian(c):
     c.run(f"kubectl run -it debian --image=debian:bullseye-slim --restart=Never -- bash")
 
 
+@invoke.task
+def clean_collectstatic(c):
```

Comment:
Does this need to get added to the ``pod`` Collection at the end of the file?

Reply:
Yes, good :eyes:! Updated now.